playground/models/networks/encoder.py

```diff
@@ -127,10 +129,22 @@ def __init__(self, data_dims, noise_dim, latent_dims, network_architecture='synt
         latent_space, features = [], []
         inputs = [Input(shape=(dim,), name="enc_data_input_{}".format(i)) for i, dim in enumerate(data_dims)]
         standard_normal_sampler = Lambda(sample_standard_normal_noise, name='enc_normal_sampler')
-        if network_architecture == 'synthetic':
+        if noise_mode == 'add':
+            standard_normal_sampler.arguments = {'seed': config['seed'], 'noise_dim': noise_dim, 'mode': 'add'}
+        elif noise_mode == 'concat':
+            if network_architecture == 'mnist':
+                assert ((noise_dim % sqrt(data_dims[0]) == 0) & (noise_dim % sqrt(data_dims[1]) == 0)), \
```

Comment:
When checking for multiple conditions it is more pythonic to use `and` or `or` instead of the bit-wise C-like `&` or `|` respectively.